### ISSUE TITLE
Make setting active item accessible

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#18644]
Lots of good info in this thread:
https://github-com/microsoft/vscode/-/166920-issuecomment-1433703251

The overarching problem is that a quick pick cannot have a placeholder _and_ set active items at the same time because the placeholder must be read out by a screenreader to be accessible.

We should try out using `aria-live` instead of `aria-activedecendant` to make this work.
